openssl/src/ssl/tests/mod.rs

```diff
@@ -226,10 +237,10 @@ run_test!(new_sslstream, |method, stream| {
     SslStream::connect(&SslContext::new(method).unwrap(), stream).unwrap();
 });
 
-run_test!(get_ssl_method, |method, _| {
-    let ssl = Ssl::new(&SslContext::new(method).unwrap()).unwrap();
-    assert_eq!(ssl.ssl_method(), method);
-});
+// run_test!(get_ssl_method, |method, _| {
```

Comment:
Ah right. Let's leave ssl_method out entirely for now and see if anyone asks for it back. The implementation always seemed a bit sketchy to me with the unwrap.